tests/src/FunctionalJavascript/Integration/AutosaveFormTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\Tests\thunder\FunctionalJavascript\Integration;
+
+use Drupal\Tests\thunder\FunctionalJavascript\ThunderFormFieldTestTrait;
+use Drupal\Tests\thunder\FunctionalJavascript\ThunderJavascriptTestBase;
+use Drupal\Tests\thunder\FunctionalJavascript\ThunderParagraphsTestTrait;
+
+/**
+ * Tests the autosave support for nodes in Thunder.
+ *
+ * @group Thunder
+ */
+class AutosaveFormTest extends ThunderJavascriptTestBase {
+
+  use ThunderFormFieldTestTrait;
+  use ThunderParagraphsTestTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    // Adjust the autosave form submission interval.
+    \Drupal::configFactory()
+      ->getEditable('autosave_form.settings')
+      ->set('interval', 2000)
+      ->save();
+  }
+
+  /**
+   * Tests the autosave functionality in an existing article.
+   */
+  public function testAutosaveInExistingEntity() {
+    $this->drupalGet('node/7/edit');
+    $page = $this->getSession()->getPage();
+
+    // Make some changes.
+    $this->setFieldValue($page, 'field_tags[]', [[5, 'Drupal'], 'Tag2']);
```

Comment:
It's not so relevant, but it's also possible to pull this into `$this->setFieldValues($page, $fieldValues);`